armi/reactor/converters/tests/test_axialExpansionChanger.py

```diff
@@ -13,34 +13,30 @@
 # limitations under the License.
 
 """Test axialExpansionChanger."""
-from statistics import mean
+# pylint: disable=missing-function-docstring,missing-class-docstring,abstract-method,protected-access
```

Comment:
Please remove this pylint.  We no longer use pylint in ARMI.

Thanks!

Reply:
https://github.com/terrapower/armi/pull/1316/commits/a39605c7e234ac63ce0ab94d4ad0e0e2966f08ba